addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -292,6 +310,22 @@ export const dateField = {
             type: "boolean",
             help: _t(`Displays a warning icon if the input dates are in the future.`),
         },
+        {
+            label: _t("Minimal precision"),
+            name: "min_precision",
+            type: "string",
```

Comment:
Sorry I didn't see it earlier but the `supportedOptions` should reflect the changes in the props, the string type is used as a selection so this should look like this:
```js
type: "selection",
choices: [
    { label: _t("Days"), value: "days" },
    { label: _t("Months"), value: "months" },
    { label: _t("Years"), value: "years" },
    { label: _t("Decades"), value: "decades" },
],
```

Reply:
It should be ok now, thank you for you review @BastienFafchamps :smile: